sofl/importer/onlinefix_source.py

```
@@ -0,0 +1,48 @@
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from sofl.importer.source import Source, SourceIterable
+
+class OnlineFixSourceIterable(SourceIterable):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)